common/tests/state/untrusted/test_state_kv.cpp

```diff
@@ -38,6 +38,24 @@ void test_state_kv() {
     size_t test_key_length = TEST_KEY_STRING_LENGTH;
     ByteArray id;
 
+//################ TEST LOG MEMORY ####################################################################################
+    try
+    {
+        LOG_MEMORY;
+        SAFE_LOG(PDO_LOG_DEBUG, "\n");
+        pstate::State_KV skv(state_encryption_key_);
```

Comment:
I can change it.
I think there is an asymmetry in SAFE_LOG:
* inside the enclave it includes `\n`
* outside the enclave it doesn't

Reply:
done

Reply:
yeah, but i think thats because safe log connects to the python logger. you can set the logging function to print a newline in your tests. that's why we have the generality.